packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -536,16 +539,19 @@ export async function* OpenAIChatModel(
 
         return AI.AppendOnlyStream;
       };
-      const assistantStream = memo(
-        <Stream {...debugRepresentation(() => `${accumulatedContent}${complete ? '' : '▮'}`)} />
+      const assistantMessage = memo(
+        <AssistantMessage>
+          <Stream {...debugRepresentation(() => `${accumulatedContent}${complete ? '' : '▮'}`)} />
+        </AssistantMessage>
       );
-      yield <AssistantMessage>{assistantStream}</AssistantMessage>;
```

Comment:
Nah, I just needed a reference to the assistant message instead of the stream and felt this was easier to read than

```tsx
<AssistantMessage>
   {memo(<Stream />)}
</AssistantMessage>
```

and in particular, I wanted to have the `render` and the `memo` call operate on the same value.